calcs/source.py

```diff
@@ -19,31 +22,117 @@ def __init__(self, m_1, m_2, f_orb, ecc, dist, ecc_tol=0.1, stat_tol=1e-2):
         m_2 : `float/array`
             secondary mass
 
-        forb : `float/array`
-            orbital frequency
-
         ecc : `float/array`
             initial eccentricity
 
         dist : `float/array`
             luminosity distance to source
 
-        ecc_tol : `float`
-            eccentricity above which a binary should be
-            considered eccentric
+        forb : `float/array`
```

Comment:
I love these changes which will make this much more flexible for just evolving sources in the case where we don't necessarily care about LISA SNRs.